### PR TITLE
added disc integration

### PR DESCRIPTION
**Pull request by IMGROOT2**
also, edited text

### CODE REVIEW

File: src/index.pug [hunk 1]
```diff
@@ -164,15 +164,13 @@ html(lang="en" style="overflow-x: hidden;")
...
-                p For any questions about the discord server contact OP#1192 or Nikhilatvocabustudy#4578 on discord.
+                p The server in completely student-run, and Nikhil, Omkar, and Siddhant do not endorse it. The server is moderation by Nikhil Gupta, Omkar Patil, and Ruhan Gupta, in addition to our team of moderators and helpers.
+                p For any questions about the Discord server, please contact "OP#1192" or "Nikhilatvocabustudy#4578" in Discord, or email #[a(href="mailto: management@vocabustudy.org") management@vocabustudy.org].
                 p If you wish to join, feel free to do so below!
-                h2.mdc-typography--headline4 
-                    a.a(href="https://discord.gg/UrRs7Bbr6d" target="_blank") Vocabustudy Official Discord Server
+                h2.mdc-typography--headline4
+                    iframe.nohuechange(src='https://discord.com/widget?id=1022663736936370349&theme=dark' width='400px' height='500px' allowtransparency='true' frameborder='0' sandbox='allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts')
```
Comment (grimsteel) [inline src/index.pug:173]: I'd rather not have an embed because it might trigger ContentKeeper eventually

**Review by grimsteel — CHANGES_REQUESTED** [review]

File: src/index.pug [hunk 2]
```diff
@@ -168,9 +168,9 @@ html(lang="en" style="overflow-x: hidden;")
                 p Have you joined our Discord server yet? There's free homework help, support, bots, and more!
                 p The server in completely student-run, and Nikhil, Omkar, and Siddhant do not endorse it. The server is moderation by Nikhil Gupta, Omkar Patil, and Ruhan Gupta, in addition to our team of moderators and helpers.
                 p For any questions about the Discord server, please contact "OP#1192" or "Nikhilatvocabustudy#4578" in Discord, or email #[a(href="mailto: management@vocabustudy.org") management@vocabustudy.org].
-                p If you wish to join, feel free to do so below!
-                h2.mdc-typography--headline4
-                    iframe.nohuechange(src='https://discord.com/widget?id=1022663736936370349&theme=dark' width='400px' height='500px' allowtransparency='true' frameborder='0' sandbox='allow-popups allow-popups-to-escape-sandbox allow-same-origin allow-scripts')
+                p If you wish to join, you can join #[a(href=https://discord.gg/UrRs7Bbr6d) here].
+                //h2.mdc-typography--headline4
```
Comment (grimsteel) [inline src/index.pug:172]: Just delete this because it's causing build failures

**Review by grimsteel — CHANGES_REQUESTED** [review]

File: src/index.pug (hunk 2 above)
Comment (grimsteel) [reply src/index.pug:172]: Oops I misread the error

Comment (IMGROOT2) [reply src/index.pug:172]: it's because i forgot the quotation marks around the link

Comment (IMGROOT2) [reply src/index.pug:172]: lol yea ur good

**Review by grimsteel — APPROVED** [review]

**Pull request merged**